market-generator/src/main/java/com/market/banica/generator/Greeting.java

```diff
@@ -0,0 +1,7 @@
+package com.market.banica.generator;
+
+public class Greeting {
```

Comment:
This class is not necessary as we have the MarketGeneratorApplication.

Reply:
if this class really needs to be here, please rename it to be understandable what it is about and the message in main method should be more descriptive as well